src/sentry/conf/server.py

```diff
@@ -1560,22 +1560,40 @@ def get_sentry_sdk_config():
 }
 
 KAFKA_PREPROCESS = 'events-preprocess'
+KAFKA_PREPROCESS_DEAD = 'events-preprocess-dead'
```

Comment:
Not in love with this but I felt dead letter topics needed to follow the same pattern of "topics have a name in code and they can be renamed/pointed at their own cluster by the end user."